data/json/items/migration.json

```diff
@@ -5,6 +5,11 @@
     "replace": "adv_UPS_off"
   },
   {
+    "id": "battery_truck",
+    "type": "MIGRATION",
+    "replace": "battery_car"
```

Comment:
Risky: there are (player-made) vehicles with truck and car batteries mounted on the same tile.